ntoskrnl/ke/i386/thrdini.c

```diff
@@ -78,7 +78,7 @@ KiThreadStartup(VOID)
     StartFrame->SystemRoutine(StartFrame->StartRoutine, StartFrame->StartContext);
 
     /* If we returned, we better be a user thread */
-    if (!StartFrame->UserThread) DbgBreakPoint();
+    if (!StartFrame->UserThread) KeBugCheck((ULONG)0x0000000EL); // NO_USER_MODE_CONTEXT
```

Comment:
You should be able to use the `NO_USER_MODE_CONTEXT` constant.

Reply:
While you're it, would you mind also updating this to the current style (no single-line ifs, always use braces)?
```
if (...)
{
    KeBugCheck(...)
}
```
Thanks.

Reply:
Added NO_USER_MODE_CONTEXT and updated to the current style,